PhotoPlusFrontend/PhotoPlus/src/app/services/user/user.service.spec.ts

```diff
@@ -1,16 +0,0 @@
-import { TestBed } from '@angular/core/testing';
-
-import { UserService } from './user.service';
-
-describe('UserService', () => {
-  let service: UserService;
-
-  beforeEach(() => {
-    TestBed.configureTestingModule({});
-    service = TestBed.inject(UserService);
-  });
-
-  it('should be created', () => {
-    expect(service).toBeTruthy();
-  });
-});
```

Comment:
why do you remove testing?

Reply:
Test is generated by Angular CLI it does not bring anything useful. We do not even run tests in our Angular for now so it is unnecessary.

Reply:
I do

Reply:
Also "the convention for Angular applications is to have a .spec.ts file for each .ts file."
Further reading:
https://angular.io/guide/testing

Besides, if you'd decided that tests are unnecessary, why did you remove only one spec file?

Reply:
I still disagree. We can keep that convention, but that make sense only if we cover code with tests which we don't. (see comments below pasted post: https://stackoverflow.com/questions/37502809/what-are-the-spec-ts-files-generated-by-angular-cli-for) 
Did You wrote some tests in that file? I did not notice any custom test for ours components. So we simply do not have test for now and in that case running test is pointless. 
For now this files are simply trashes. And as I presume we won't be covering frontend with tests which is not a good practice and I am aware of it, but pretending that we test with generated files?
Test created during file generation are not checking any of created functionality in component but only creation and as I see component compilation. So as w can presume this is done by simply running ng build or running application. Maybe I'm wrong. I'm not saying that that my pov is right but still worth considering.

```
Besides, if you'd decided that tests are unnecessary, why did you remove only one spec file?
```
As You can see thats not a scope of this PR so for now I did it only for modified services.